tests/plugins/justify/manual/justify.html

```diff
@@ -0,0 +1,14 @@
+<div id='editor'>
```

Comment:
I make name `justifyrespectacf` it should be now more specific name of the test.